### PR TITLE
Remove internal deprecations.

### PR DESCRIPTION
**Pull request by nemanja-boric-sociomantic**
This PR removes references to deprecated methods in swarm from unittests, so swarm doesn't hit it's own deprecations.

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
Yeah, I was going to say, why undeprecate it when we can just not call it internally. I see you updated ;)

**Comment by nemanja-boric-sociomantic** [comment]
Well, the `protos` still need to call it. However, I forgot that we can do `static if` on the swarm version inside protos.

**Comment by gavin-norman-sociomantic** [comment]
The protos need to call it because in older swarm versions it used to do something?

**Comment by nemanja-boric-sociomantic** [comment]
Yes, in older swarm versions, TCP_NODELAY was not turned on by default and we thought that's too much for a patch release.

File: src/swarm/neo/client/mixins/ClientCore.d [hunk 1]
```diff
@@ -239,7 +239,6 @@ template ClientCore ( )
             settings.conn_notifier, settings.auto_connect);
```
Comment (gavin-norman-sociomantic) [inline src/swarm/neo/client/mixins/ClientCore.d:239]: >This method is de-facto deprecated and does nothing.

Mini comment: this is wrong. The method is _actually_ deprecated.

Comment (nemanja-boric-sociomantic) [reply src/swarm/neo/client/mixins/ClientCore.d:239]: Thanks, updated.

**Review by gavin-norman-sociomantic — APPROVED** [review]

**Pull request merged**